monitors/new_end_of_run_monitor.py

```diff
@@ -60,6 +60,39 @@ def get_prefix_zeros(run_number_str):
     return zeros
 
 
+def is_integer(int_str):
```

Comment:
In a separate issue later on we should refactor this and other similar functions to somewhere that more generic so that other areas of the code base can potentially make use of very generic functions such as this